ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml

```diff
@@ -53,6 +55,8 @@ StackView {
     QtObject {
         id: d
 
+        property bool isOnlyAdmin: root.isAdmin && !root.isOwner && !root.isTokenMasterOwner
```

Comment:
`readonly`

Reply:
And, I'd rename this to `isAdminOnly`

`isOnlyAdmin` means "I am the only admin here", and not "I am just an admin, and not an owner or token master"

Reply:
Ok, got it!!! Will update